spec/models/partner_spec.rb

```diff
@@ -291,6 +291,7 @@
     let(:contact_name) { "Jon Ralfeo" }
     let(:contact_email) { "jon@entertainment720.com" }
     let(:contact_phone) { "1231231234" }
+    let(:notes) { "Lorem ipsum" }
 
```

Comment:
This is a very  minor quibble -- If I'm explicitly setting a value,  I would set it to something other than the default from the factory.

Reply:
Fixed it here [bed87a8](https://github.com/rubyforgood/human-essentials/pull/4195/commits/bed87a86ff996a12395bb97fbc8051e44743571d)